test/functional/apps/discover/_collapse_expand.js

```diff
@@ -18,21 +20,23 @@ import {
         var fromTime = '2015-09-19 06:31:44.000';
         var toTime = '2015-09-23 18:31:44.000';
 
-        // start each test with an empty kibana index
-        return scenarioManager.reload('emptyKibana')
+        // delete .kibana index
+        return esClient.delete('.kibana')
+        .then(function () {
+          // wait for Kibana to re-create it and add UTC and defaultIndex
+          return common.try(function () {
```

Comment:
There's actually a couple of different cases now.  

1. For Settings tests, we delete the .kibana index
 and then call `getConfigId()` just as a means to wait for it to be re-created.

2. For Discover and Visualize tests, we delete the .kibana index and then call `esClient.updateConfigDoc({'dateFormat:tz':'UTC', 'defaultIndex':'logstash-*'})` (which calls `getConfigId()`)
and load the logstash index pattern with elasticDump.

3. For Dashboard tests, we delete the .kibana index and then call `esClient.updateConfigDoc({'dateFormat:tz':'UTC', 'defaultIndex':'logstash-*'})`
and load the logstash index pattern and saved visualizations with elasticDump.

Maybe a `deleteAndUpdate(index, configDocMap)` method which deletes the index, and gets the configId, and optionally does the configDocMap if it's passed in.